--- lib/WebGUI/Session.pm	Fri Feb 14 06:00:45 2003
+++ lib/WebGUI/Session.pm-mis	Tue Feb 18 13:53:28 2003
@@ -409,7 +409,7 @@
                 }
         } else {
                 $config = new Data::Config
-		$session{config}{webguiRoot}.'/etc/'.$session{config}{configFile};
+		$session{config}{configFile};
                 foreach ($config->param) {
                         $session{config}{$_} = $config->param($_);
                 }
